gtsam/hybrid/HybridGaussianFactorGraph.cpp

```diff
@@ -428,35 +424,9 @@ EliminateHybrid(const HybridGaussianFactorGraph &factors,
   }
 }
 
-/* ************************************************************************ */
```

Comment:
I was super doubtful when these were first added since `GaussianFactorGraph` and `DiscreteFactorGraph` don't have these, but I guess that PR was merged before I could question this.

Glad to see we're cutting down on the sugar. 😉